From f13b59663b779376e5dc3889179bbd0ca79585d2 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Fri, 26 Sep 2014 10:34:46 +0100
Subject: Remove .MIPS.abiflags section from images

Recent versions of binutils generate this section, but we don't need it
when producing raw images.

Bug-Debian: https://bugs.debian.org/762307
Forwarded: no
Last-Update: 2014-09-26

Patch-Name: mips-abiflags.patch
---
 gentpl.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gentpl.py b/gentpl.py
index bdcae1a..fe76d49 100644
--- a/gentpl.py
+++ b/gentpl.py
@@ -753,7 +753,7 @@ def image(defn, platform):
 if test x$(TARGET_APPLE_LINKER) = x1; then \
   $(MACHO2IMG) $< $@; \
 else \
-  $(TARGET_OBJCOPY) $(""" + cname(defn) + """_OBJCOPYFLAGS) --strip-unneeded -R .note -R .comment -R .note.gnu.build-id -R .reginfo -R .rel.dyn -R .note.gnu.gold-version $< $@; \
+  $(TARGET_OBJCOPY) $(""" + cname(defn) + """_OBJCOPYFLAGS) --strip-unneeded -R .note -R .comment -R .note.gnu.build-id -R .reginfo -R .rel.dyn -R .note.gnu.gold-version -R .MIPS.abiflags $< $@; \
 fi
 """)
 
